packages/docs/src/examples/transitions/misc-expand-x-padding.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-app>
+    <v-container>
+      <v-btn class="ma-2" color="primary" @click="expand = !expand">
+        Expand Transition
+      </v-btn>
+
+      <v-expand-transition>
```

Comment:
That `*-x-*` transition works just fine when alert is meant to extend to 100%, but requires more CSS skills to get something that fits the content. It might be helpful for the devs to see an example right here and learn it is possible with this little trick:

```html
<v-expand-x-transition>
  <v-responsive v-show="expand" width="fit-content">
    <v-alert
      text="Smooth transition here"
      type="info"
      class="text-no-wrap"
    ></v-alert>
  </v-responsive>
</v-expand-x-transition>
```